frigate/config.py

```diff
@@ -1077,6 +1078,14 @@ def runtime_config(self, plus_api: PlusApi = None) -> FrigateConfig:
             config.mqtt.user = config.mqtt.user.format(**FRIGATE_ENV_VARS)
             config.mqtt.password = config.mqtt.password.format(**FRIGATE_ENV_VARS)
 
+        # set default min_score for object attributes
+        for attribute in ALL_ATTRIBUTE_LABELS:
+            if (
+                not config.objects.filters
+                or config.objects.filters.get(attribute, FilterConfig()).min_score == 0.5
```

Comment:
Won't this overwrite all their filters if they had specified a min_score of 0.5?

Reply:
yeah, if the filter was manually set to 0.5 it would get overwritten. The issue I am trying to figure out how to handle is the case where a user sets `min_area` for example but the default min_score shouldn't be affected by that

Reply:
So I'm basically assuming users won't want to set the min_score for an attribute to 0.5, there definitely may be a better approach but I can't find a good way to have the default_factory for a field be able to access the key from the dict for example. Because if we knew the label for that particular FilterConfig then the min_score could be automatic